controllers/watcher_controller.go

```diff
@@ -728,7 +741,7 @@ func (r *WatcherReconciler) generateServiceConfigDBSync(
 		"APIPublicPort": fmt.Sprintf("%d", watcher.WatcherPublicPort),
 	}
 
-	return GenerateConfigsGeneric(ctx, helper, instance, envVars, templateParameters, customData, labels, false)
+	return GenerateConfigsGeneric(ctx, helper, instance, envVars, templateParameters, customData, labels, true)
```

Comment:
why are we changing the boolen here? I don't see any other change for DBSync

Reply:
I see, I understand now, but I did not find it obvious from a first pass, thanks @amoralej